nuxeo-importer-core/src/main/resources/en_US.dic

```diff
@@ -1127,14 +1127,14 @@ Astrix/M
 AstroTurf/S
 Astroturf/M
 Asturias/M
-Asunci�n/M
+Asunci�n/M
```

Comment:
Why do you we have these changes ?

Reply:
I don't know, the eclipse was OK, I'm checking